docs/test-plan.md

```diff
@@ -143,6 +143,39 @@ Run one of the above MRVAs, but cancel it from within VS Code:
 - Check that the workflow run is also canceled.
 - Check that any available results are visible in VS Code.
 
+### CodeQL Model Editor
+
+Note: this is still behind feature flags.
+
+#### Test Case 1: Opening the model editor
+
+1. Download the `dsp-testing/sql2o-example` database from GitHub.
+2. Open the Model editor with Cmd+P "Open CodeQL Model Editor".
+   - Check that the editor loads and shows methods to model.
+   - Check that methods are grouped per library.
+   - Check that the "Open database" link works.
+   - Check that the "Open extension pack" link works.
+
+#### Test Case 2: Model methods
+
+1. Expand one of the packages
+   - Change the model type and check that the other dropdowns change.
+2. Save the modeled methods.
+   - Check that the model `yml` file has the entries that was modeled.
+
+#### Test Case 3: Model with AI
+
+1. Click "Model with AI".
+   - Check that rows change to "Thinking".
+   - Check that results come back and rows get filled out.
+
+#### Test Case 4: Model as dependency
+
+1. Click "Model as dependency"
+   - Check that grouping are now per package.
```

Comment:
I wasn't sure how to check this. Does "per package" just mean that it looks like `org.example` instead of `example@library`?

Reply:
Yes. I'll try to add some more text here.